viewer/src/main/webapp/viewer-html/common/ajax/FeatureInfo.js

```diff
@@ -60,8 +60,12 @@ Ext.define("viewer.FeatureInfo", {
                 successFunction(response);
             },
             failure: function(result) {
-                if(failureFunction != undefined) {
-                    failureFunction("Ajax request failed with status " + result.status + " " + result.statusText + ": " + result.responseText);
+                if (failureFunction !== undefined) {
+                    // -1 means request aborted; this is not an error, but user initiated
+                    // 0 is a communication error (timeout)
+                    if (result.status > -1) {
```

Comment:
@mtoonen Not sure I like this, but it works for the tooltip as that creates a bunch of aborted requests